pr-scripts/create_pr.sh

```diff
@@ -22,20 +22,33 @@ REPO="$1"
 OLD_TAG="$2"
 NEW_TAG="$3"
 FILEPATH="$4"
+DRY_RUN_FLAG="$5"
+
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
 
 if [ $REPO = "eks-distro-build-tooling" ]; then
     CHANGED_FILE="Tag file"
 elif [ $REPO = "eks-distro" ]; then
-    CHANGED_FILE="Makefile"
+    CHANGED_FILE="Makefiles"
+elif [ $REPO = "eks-distro-prow-jobs" ]; then
+    CHANGED_FILE="Prow jobs"
+fi
+
+COMMIT_MESSAGE="[PR BOT] Update EKS Distro base image tag"
+if [ $REPO = "eks-distro-prow-jobs" ]; then
+    COMMIT_MESSAGE="[PR BOT] Update builder-base image tag in Prow jobs"
 fi
 
 PR_TITLE="Update base image tag in ${CHANGED_FILE}"
-sed -i '' "s,CHANGED_FILE,${CHANGED_FILE}," eks-distro-base/pr_body
-PR_BODY=$(cat pr_body)
+sed -i "s,CHANGED_FILE,${CHANGED_FILE}," ${REPO_ROOT}/../pr-scripts/eks_distro_base_pr_body
+PR_BODY=$(cat ${REPO_ROOT}/../pr-scripts/eks_distro_base_pr_body)
+if [ $REPO = "eks-distro-prow-jobs" ]; then
+    PR_BODY=$(cat ${REPO_ROOT}/../pr-scripts/builder_base_pr_body)
+fi
 
-cd ../${REPO}
+cd ${REPO_ROOT}/../../../eks-distro-pr-bot/${REPO}
 git config user.name "EKS Distro PR Bot"
-git remote add upstream git@github.com:aws/${REPO}.git
+git remote add upstream https://github.com/${REPO_OWNER}/${REPO}.git
 git remote add origin git@github.com:eks-distro-pr-bot/${REPO}.git
```

Comment:
Not sure if this will cause a problem. Let's try pushing to a separate branch based on repo_owner. If we can't talk to `eks-distro-pr-bot` fork, then we can have it run against our own fork if repo_owner is `aws`.